app/assets/javascripts/bastion/bastion.module.js

```diff
@@ -76,18 +76,27 @@ angular.module('Bastion').config(
  * @requires $breadcrumb
  * @requires PageTitle
  * @requires BastionConfig
+ * @requires repositoryTypes
  *
  * @description
  *   Set up some common state related functionality and set the current language.
  */
-angular.module('Bastion').run(['$rootScope', '$state', '$stateParams', 'gettextCatalog', 'currentLocale', '$location', '$window', '$breadcrumb', 'PageTitle', 'BastionConfig',
-    function ($rootScope, $state, $stateParams, gettextCatalog, currentLocale, $location, $window, $breadcrumb, PageTitle, BastionConfig) {
+angular.module('Bastion').run(['$rootScope', '$state', '$stateParams', 'gettextCatalog', 'currentLocale', '$location', '$window', '$breadcrumb', 'PageTitle', 'BastionConfig', 'repositoryTypes',
+    function ($rootScope, $state, $stateParams, gettextCatalog, currentLocale, $location, $window, $breadcrumb, PageTitle, BastionConfig, repositoryTypes) {
         var fromState, fromParams, orgSwitcherRegex;
 
         $rootScope.$state = $state;
         $rootScope.$stateParams = $stateParams;
         $rootScope.transitionTo = $state.transitionTo;
         $rootScope.$location = $location;
+        $rootScope.$repositoryTypes = repositoryTypes;
```

Comment:
I know I told you on IRC that I was fine with this but now that I see it I'm having a bad reaction.  I know that all of the katello angular stuff is deprecated but I wonder if we should set this precedent. 

Using a service and then injecting that service into each controller is more work but looking at https://github.com/Katello/katello/pull/7204 there are only a handful of places where this is necessary.  When this change was described to me I thought there were going to be many more places where the DI would be necessary.

Reply:
yeah, it ended up being fewer places than i intiailly thought.  I'm okay moving it to a service.

Reply:
Given the above, I'd vote service too. I agree if this is deprecated code we don't need to care as much but given we still have to maintain it we don't want it to fall into complete disarray given bugs can pop up from anywhere.